stable/osg-hosted-ce/osg-hosted-ce/templates/policy.yaml

```
@@ -0,0 +1,38 @@
+{{ if .Values.NetworkPolicy.Enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ template "osg-hosted-ce.fullname" . }}
+  labels:
+    app: {{ template "osg-hosted-ce.name" . }}
+    chart: {{ template "osg-hosted-ce.chart" . }}
+    release: {{ .Release.Name }}
+    instance: {{ .Values.Instance | quote }}
+spec:
+  podSelector:
+    matchLabels:
+      app: {{ template "osg-hosted-ce.name" . }}
+      chart: {{ template "osg-hosted-ce.chart" . }}
+      release: {{ .Release.Name }}
+      instance: {{ .Values.Instance | quote }}
+  policyTypes:
+  - Ingress
+  - Egress
+  egress:
+  - to:
+    - ipBlock:
+        cidr: 0.0.0.0/0
```

Does this work? We just do the following:

```suggestion
  - {}
```

---

Both approaches will work

https://kubernetes.io/docs/concepts/services-networking/network-policies/#default-allow-all-egress-traffic